samples/cellular/modem_shell/src/sock/sock.c

```diff
@@ -354,7 +354,8 @@ static int sock_set_tls_options(
 	uint32_t sec_tag,
 	bool session_cache,
 	int peer_verify,
-	char *peer_hostname)
+	char *peer_hostname,
+	int dtls_connection_id)
```

Comment:
You could rename all `dtls_connection_id`s to `dtls_cid`. I think it would be clearer to align the option and variable names.